src/main/java/roomescape/login/AuthController.java

```diff
@@ -0,0 +1,60 @@
+package roomescape.login;
+
+import jakarta.servlet.http.Cookie;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import java.util.Arrays;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import roomescape.member.MemberResponse;
+
+@RestController
+public class AuthController {
+    private final AuthService authService;
+
+    public AuthController(AuthService authService) {
+        this.authService = authService;
+    }
+
+    @PostMapping("/login")
+    public ResponseEntity<Void> login(@RequestBody AuthRequest authRequest, HttpServletResponse response) {
+        AuthResponse authResponse = authService.login(authRequest);
+
+        Cookie cookie = new Cookie("token", authResponse.token());
+        cookie.setHttpOnly(true);
+        cookie.setPath("/");
+        response.addCookie(cookie);
+
+        return ResponseEntity.ok().build();
+    }
+
+    @GetMapping("/login/check")
+    public ResponseEntity<MemberResponse> checkLogin(HttpServletRequest request) {
+        Cookie[] cookies = request.getCookies();
+
+        if (cookies == null) {
+            throw new IllegalArgumentException("로그인을 해주세요.");
+        }
+
+        String token = extractTokenFromCookie(cookies);
+
+        if (token.isEmpty()) {
+            throw new IllegalArgumentException("로그인을 해주세요.");
+        }
+
+        MemberResponse memberResponse = authService.checkLogin(token);
+
+        return ResponseEntity.ok(memberResponse);
+    }
+
+    private String extractTokenFromCookie(Cookie[] cookies) {
+        return Arrays.stream(cookies)
+                .filter(cookie -> cookie.getName().equals("token"))
+                .map(Cookie::getValue)
+                .findFirst()
+                .orElse("");
+    }
```

Comment:
```suggestion
    private String extractTokenFromCookie(Cookie[] cookies) {
        return Arrays.stream(cookies)
                .filter(cookie -> cookie.getName().equals("token"))
                .map(Cookie::getValue)
                .findFirst()
                .orElseThrow(() -> new IllegalArgumentException("로그인을 해주세요."));
    }
```

이러면 `checkLogin()`에서 if문 하나 없앨 수 있겠네요

Reply:
오.. 그렇네요! 수정해보겠습니다~~